nautobot/ui/src/views/generic/ObjectRetrieve.js

```diff
@@ -175,18 +181,22 @@ export default function ObjectRetrieve({ api_url }) {
     // Object Data
     const {
         data: objectData,
-        isError: error,
+        error: objectDataError,
         isLoading: objectDataLoading,
     } = useSWR(() => api_url, fetcher);
     const ui_url = objectData?.url
-        ? `${objectData.url}detail-view-config/`
+        ? `${objectData.url}detail-view-onfig/`
         : null;
-    var { data: appConfig } = useSWR(() => ui_url, fetcher);
+    const {
+        data: appConfig,
+        error: appConfigError,
+        isLoading: appConfigLoading,
+    } = useSWR(() => ui_url, fetcher);
```

Comment:
Can/should we be using `getRESTAPI` instead of `useSWR` here? Since it was already recently updated to handle 4xx errors itself, it feels weird to be having to add equivalent logic in this file instead of just using that. What am I missing? Is this something that we can't change until we do #3526 and move it to an OPTIONS query?

Reply:
Yeah that and also `useGetRESTAPIQuery` right now does not have the capability of adding url `query params` or just straight out specifying an `url` to get the information from. So we are still using `useSWR()` here for `DetailViewConfig`, `ChangeLog` and `Notes`.